spec/acceptance/1_domain_mode_cluster_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'spec_helper_acceptance'
+
+# This check needs to be here or Beaker will try to run find_only_one on
+# non-domain-mode tests and fail miserably.
+if RSpec.configuration.keycloak_domain_mode_cluster
+  describe 'keycloak domain mode cluster' do
```

Comment:
This should be

```
describe 'keycloak domain mode cluster' if: RSpec.configuration.keycloak_domain_mode_cluster
```

One fewer level of spacing for everything in this file.

Reply:
I had to revert the change you proposed here. As mentioned in the comments the if statement needs to be where it is - otherwise the find_only_one functions gets triggered even on non-domain-mode nodesets. I don't fully understand what causes that behavior.